installer/scripts/omsadmin.sh

```diff
@@ -608,6 +617,12 @@ remove_all()
     # Remove LAD workspace
     WORKSPACE_ID="LAD"
     remove_workspace
+
+    # remove scom workspace
+    if [ `ls -1 $ETC_DIR | grep scom | wc -l` -eq 1 ]; then
```

Comment:
This is susceptible to the same issues as `httpsport`, above. `egrep` would make this trivial, but I wouldn't care suggest that for OMS, we already have too many problems with programs not being installed. In fact, it wouldn't be a bad idea to see what programs our installation scripting for `omi` and `SCXcore` uses, and at least make sure that Laura is aware of that.

Is there some parser in OMS that we can call that can tell us if the scom workspace exists? Or I suppose you could have a little bash function that enumerated files in `$ETC_DIR`, and returned success if it found the file you're looking for. Bash can do that, and this need not be portable because OMS only runs on Linux.

I guess it all depends on how bullet-proof you want this to be ...